### PR TITLE
fix code stype format

### PR DESCRIPTION
**Pull request by zwwhdls**
fix https://github.com/kubernetes/website/issues/15144

### CODE REVIEW

File: content/zh/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 1]
```diff
@@ -34,48 +34,37 @@ content_template: templates/task
...
-    
-    ```shell
-    kubectl create -f https://k8s.io/docs/tasks/inject-data-application/commands.yaml
-    ```
+   ```shell
+   kubectl create -f https://k8s.io/docs/tasks/inject-data-application/commands.yaml
+   ```
  
-<!--
 2. List the running Pods:
```
Comment (idealhack) [inline content/zh/docs/tasks/inject-data-application/define-command-argument-container.md]: remove this line as well, otherwise LGTM

**Comment by idealhack** [comment]
better to follow the style of English version: https://raw.githubusercontent.com/kubernetes/website/master/content/en/docs/tasks/inject-data-application/define-command-argument-container.md

**Comment by idealhack** [comment]
was this fixed? let me know if you have any problems :)

**Comment by zwwhdls** [comment]
@idealhack Hi, I saw the pr https://github.com/kubernetes/website/pull/15268 has been merged for this [issue](https://github.com/kubernetes/website/issues/15144) , although that pr is created after this one. So I close this pr, sad.

**Comment by idealhack** [comment]
@zwwhdls oh... ok, feel free to find more reviewers on slack next time! thank you anyway!

**Pull request closed without merging**